run-dlp/workflow-definitions/dlp-gcs-file.yaml

```diff
@@ -0,0 +1,65 @@
+main:
+    params: [args]
+    steps:
+    - initVariables:
+        assign:
+        - project: 'uri-test'
+        - topic: 'workflows-demo'
+        - timeInterval: '10'
+    - Create DLP Job:
+        call: http.post
+        args:
+            url: https://us-central1-uri-test.cloudfunctions.net/DlpInspectGcsFile
```

Comment:
I try to avoid hard-coding URLs in my workflow definitions, as they are not portable across projects.
One approach could be to pass the URL of that function as a workflow argument.

Reply:
I agree on this comment. 
The issue here is that the workflow is triggered by a file uploaded to GCS so the input is whatever is sent from GCS.

Reply:
Surely, the Cloud Functions URL is not set by GCS event, is it? Hardcoding the URL is OK but people won't be able to run the sample out of the box. Another approach is to take the url as an argument and pass that in during that execution. That's what we do here as an example: https://github.com/GoogleCloudPlatform/serverless-photosharing-workshop/blob/master/workflows/workflows.yaml